src/sentry/integrations/message_builder.py

```diff
@@ -85,16 +90,22 @@ def build_attachment_text(group: Group, event: GroupEvent | None = None) -> Any
     ev_metadata = obj.get_event_metadata()
     ev_type = obj.get_event_type()
 
-    if ev_type == "error":
+    if not event:
+        event = group.get_latest_event()
+
+    if hasattr(event, "occurrence") and event.occurrence is not None:
+        if event.occurrence.evidence_display is not None:
```

Comment:
You should be able to just rely on `important_evidence_display` without this check